Makefile

```diff
@@ -57,11 +57,11 @@ docs:
 pb_generate:
 	@echo "Generates pb files."
 	$(PIPENV_CMD) run python -m grpc_tools.protoc \
-		--proto_path=./proto \
-		--mypy_out=./proto \
+		--proto_path=./src/pyspiffe/proto/spiffe \
+		--mypy_out=./src/pyspiffe/proto/spiffe \
 		--proto_path=. \
```

Comment:
Is the second "--proto_path=..." really needed? I executed a test without it and could generate the python files successfully.

Reply:
Good catch, that is definitely not needed.

Reply:
Yes, is not necessary. And while checking that I found I need to make another change so thanks for pointing it out.